### PR TITLE
Update browser stats and match autoprefixer settings

### PR DESCRIPTION
**Pull request by kdzwinel**
Style guide size went from 50kb to 52kb (no gzip)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,16 +34,16 @@ We support the following browsers:
...
+| Android Browser | 4.0+ | 6.69% |
+| Firefox | 42+ | 6.92% |
+| YaBrowser | 15+ | 9.71% |
+| Opera Mini | 7+ | 2.22% |
+| Opera | 33+ | 4.27% |
+| Internet Explorer | 11+ | 3.42% |
+
+Last updated: 13.01.2016
 
 Note: You can find all recent stats in [this document](https://docs.google.com/spreadsheets/d/1n0tGV042xnrcdaZjNi1UWFcp77FUf5RDgHrTBon7Kgk/) (only company accessible!)
```
Comment (kdzwinel) [inline README.md]: Karolina updated the doc, you can check it out

File: README.md [hunk 2]
```diff
@@ -34,16 +34,16 @@ We support the following browsers:
...
-| Opera | 31+ | 4.51% |
-| Internet Explorer | 11+ | 2.37% |
-
-Last updated: 14/10/2015
+| Google Chrome | 28+ | 47.53% |
+| Safari | 7+ | 10.89% |
+| Android Browser | 4.0+ | 6.69% |
+| Firefox | 42+ | 6.92% |
+| YaBrowser | 15+ | 9.71% |
+| Opera Mini | 7+ | 2.22% |
```
Comment (kdzwinel) [inline README.md]: huge traffic drop on opera mini :(

File: scripts/tasks/sass-build.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ module.exports = function (gulp, plugins, consts) {
             .pipe(plugins.sass({outputStyle: 'compressed'})
                 .on('error', plugins.sass.logError))
             .pipe(plugins.autoprefixer({
-                browsers: ['last 2 versions', 'ie 8', 'ie 9'],
+                browsers: ['last 2 versions', 'OperaMini >= 5', 'Android >= 4', 'Chrome >= 28', 'Safari >= 7'],
```
Comment (kdzwinel) [inline scripts/tasks/sass-build.js]: This translates to:

android 46
android 4.4.3-4.4.4
android 4.4
android 4.2-4.3
android 4.1
android 4
chrome 47
chrome 46
chrome 45
chrome 44
chrome 43
chrome 42
chrome 41
chrome 40
chrome 39
chrome 38
chrome 37
chrome 36
chrome 35
chrome 34
chrome 33
chrome 32
chrome 31
chrome 30
chrome 29
chrome 28
edge 13
edge 12
firefox 43
firefox 42
ie 11
ie_mob 11
ie_mob 10
ios_saf 9.0-9.2
ios_saf 8.1-8.4
op_mini 5.0-8.0
opera 34
opera 33
safari 9
safari 8
safari 7.1
safari 7

**Comment by aju** [comment]
OK

File: README.md (hunk 2 above)
Comment (vergilius) [inline README.md]: what period of time did you check?

**Comment by vergilius** [comment]
:+1:

File: README.md (hunk 2 above)
Comment (kdzwinel) [inline README.md]: Last 30 days

**Pull request merged**